base/applications/games/winmine/main.c

```diff
@@ -1092,7 +1092,7 @@ int WINAPI wWinMain( HINSTANCE hInst, HINSTANCE hPrevInst, LPWSTR cmdline, int c
     wc.cbWndExtra = 0;
     wc.hInstance = hInst;
     wc.hIcon = LoadIconW( hInst, MAKEINTRESOURCEW(IDI_WINEMINE) );
-    wc.hCursor = LoadCursorW( 0, (LPWSTR)IDI_APPLICATION );
+    wc.hCursor = LoadCursorW( 0, (LPWSTR)IDC_ARROW );
```

Comment:
For 3rd-party code(s), use preprocessor:
```suggestion
#ifndef __REACTOS__
    wc.hCursor = LoadCursorW( 0, (LPWSTR)IDI_APPLICATION );
#else
    wc.hCursor = LoadCursorW(NULL, (LPWSTR)IDC_ARROW);
#endif
```

Reply:
WineMine is not maintained in Wine anymore, so it will no longer be synced in ReactOS. This preprocessor is only used to track the diffs during the Wine syncing, and it is not used for any other purpose.
But from another side, since this is actually a Wine code, and ifdefs were previously added during ROS-specific changes, then I will add them here too.

Reply:
Done.